src/ui/shared/forms/SuccessState/SuccessStateLoader.tsx

```diff
@@ -0,0 +1,317 @@
+        alignItems: 'center',
+      }}
+    >
+      {playExitAnimation ? null : (
+        <ArrowRight
+          style={{ color: 'var(--neutral-500)', position: 'relative' }}
+        />
+      )}
+      <animated.div
+        style={{
+          ...itemsStyle,
+          position: 'absolute',
+          inset: '0 0 0 0',
+          transformOrigin: 'center',
+        }}
+      >
+        <animated.div
+          style={{
+            ...startItemStyle,
+            position: 'absolute',
+            inset: '0 0 0 0',
+            zIndex: playExitAnimation ? 1 : 0,
+          }}
+        >
+          {startItem}
+        </animated.div>
+        <animated.div
+          style={{
+            ...endItemStyle,
+            position: 'absolute',
+            inset: '0 0 0 0',
+            zIndex: playExitAnimation ? 0 : 1,
+          }}
+        >
+          {endItem}
+        </animated.div>
+      </animated.div>
+      <animated.div
+        style={{
+          ...resultIconStyle,
+          position: 'absolute',
+          inset: '0 0 0 0',
+          transformOrigin: 'center',
+        }}
+      >
+        {status === 'failed' ? (
+          <ErrorIcon />
+        ) : status === 'confirmed' ? (
+          <SuccessIcon />
+        ) : null}
+      </animated.div>
+    </div>
+  );
+}
+
+function AnimatedDots() {
+  return (
+    <HStack gap={2} className={styles.dots}>
+      <UIText kind="headline/hero">.</UIText>
+      <UIText kind="headline/hero">.</UIText>
+      <UIText kind="headline/hero">.</UIText>
+    </HStack>
+  );
+}
+
+function BackgroundDecoration({ color }: { color: string | null }) {
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        top: '-30vh',
+        left: 0,
+        right: 0,
+        height: '60vh',
+        transitionDuration: '1s',
+        willChange: 'opacity',
+        opacity: color ? 0.7 : 0,
+        background: `radial-gradient(circle at top center, ${color}, transparent 65%)`,
+      }}
+    />
+  );
+}
+
+export function SuccessStateLoader({
+  startItem,
+  endItem,
+  status,
+  pendingTitle = 'Pending',
+  confirmedTitle = 'Done',
+  failedTitle = 'Failed',
+  dropppedTitle = 'Dropped',
+  explorerUrl,
+  error,
+  confirmedContent,
+  onDone,
+}: {
+  startItem: React.ReactNode;
+  endItem: React.ReactNode;
+  status: ClientTransactionStatus;
+  pendingTitle?: string;
+  confirmedTitle?: string;
+  failedTitle?: string;
+  dropppedTitle?: string;
+  explorerUrl?: string;
+  error?: string;
+  confirmedContent?: React.ReactNode;
+  onDone?: () => void;
+}) {
+  const showLongWaitNotice = useRenderDelay(5000);
+
+  const backgroundColor =
+    status === 'confirmed'
+      ? 'var(--positive-500)'
+      : status === 'failed' || status === 'dropped'
+      ? 'var(--negative-500)'
+      : null;
+
+  return (
+    <PageColumn
+      style={{
+        width: 'clamp(320px, 100vw, 450px)',
+        marginInline: 'auto',
+        position: 'relative',
+      }}
+    >
```

Comment:
What do you think about making this whole element appear with a quick fade-in? I think currently it feels a bit off that it appears kinda unexpectedly but then has fluid animations within